parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -65,7 +65,7 @@ if [ -f $VHD_LOGS_FILEPATH ]; then
     export -f cleanUpContainerImages
     export KUBERNETES_VERSION
     echo "start to clean up container images"
-    bash -c cleanUpContainerImages &
```

Comment:
why do we need this change?

Reply:
reverting this change for this pr.
I realized that `cleanUpContainerImages` was throwing "docker.sock" not found errors as it tries to `docker rmi` some of our pre-baked images after we switch out docker for containerd. I'll fix this in a separate PR.